README.md

```diff
@@ -42,12 +42,28 @@ client.quota.callback.static.kafka.admin.ssl.truststore.location=/tmp/trust.jks
 
 # Optional list of principals not to be subjected to the quota
 client.quota.callback.static.excluded.principal.name.list=principal1,principal2
 ```
 
+### Properties and their defaults
+
+| Property                                                                  | Type    | Default                 | Description                                                                                                          |
+|---------------------------------------------------------------------------|---------|-------------------------|----------------------------------------------------------------------------------------------------------------------|
+| client.quota.callback.static.produce                                      | Double  | 1.7976931348623157e+308 | Produce bandwidth rate quota (in bytes)                                                                              |
+| client.quota.callback.static.fetch                                        | Double  | 1.7976931348623157e+308 | Consume bandwidth rate quota (in bytes)                                                                              |
+| client.quota.callback.static.request                                      | Double  | 1.7976931348623157e+308 | Request processing time quota (in seconds)                                                                           |
+| client.quota.callback.static.excluded.principal.name.list                 | List    | []                      | List of principals that are excluded from the quota                                                                  |
+| client.quota.callback.static.storage.check-interval                       | Integer | 60                      | Interval between storage check runs (in seconds, default of 0 means disabled                                         |
+| client.quota.callback.static.storage.per.volume.limit.min.available.bytes | Long    | None                    | Stop message production if availableBytes <= this value                                                              |
+| client.quota.callback.static.storage.per.volume.limit.min.available.ratio | Double  | None                    | Stop message production if availableBytes / capacityBytes <= this value                                              |
+| client.quota.callback.static.throttle.factor.fallback                     | Double  | 1.0                     | Fallback throttle factor to apply if current factor expires                                                          |
+| client.quota.callback.static.throttle.factor.validity.duration            | String  | PT5M                    | How long a throttle factor derived from a successful observation of the cluster should be applied (iso8601 duration) |
+| client.quota.callback.static.kafka.admin.bootstrap.servers                | String  | None                    | Bootstrap servers of Kafka cluster                                                                                   |
```

Comment:
The bootstrap.servers prop is currently required, maybe add that to the description. If it's not set then the broker fails to start with error:
```
org.apache.kafka.common.config.ConfigException: Missing required configuration "client.quota.callback.static.kafka.admin.bootstrap.servers" which has no default value.
```